KOA/clanwars/api.py

```diff
@@ -0,0 +1,33 @@
+class HabiticaApi(habitica.api.Habitica):
+    def __init__(self, user_id, api_token):
+        headers = {
+            AUTH_URL: "https://habitica.com",
+            #default.AUTH_HEADER_CLIENT: user_id,
+            #default.AUTH_HEADER_TOKEN: api_token
+
+            AUTH_HEADER_CLIENT: "54333f60-395f-42d0-8461-f5b59080498e",
+            AUTH_HEADER_TOKEN: "baa8235b-15f7-4a6b-a5a4-c71a2dc96a12"
```

Comment:
Please remove these from the history.
See: https://help.github.com/articles/removing-sensitive-data-from-a-repository/

Reply:
Also, email habitica about changing your api token if not already done :)

Reply:
dammit i thought i got rid of that. it's only been up for a few hours tho hopefully not too bad